packages/build/tests/plugins/tests.js

```diff
@@ -110,25 +110,6 @@ test('Validate --node-path unsupported version does not fail when no plugins are
   t.snapshot(normalizeOutput(output))
 })
 
-test('Validate --node-path version is supported by the plugin', async (t) => {
-  const systemLog = await tmp.file()
-
-  const nodePath = getNodePath('16.14.0')
-  const output = await new Fixture('./fixtures/engines')
-    .withFlags({
-      nodePath,
-      featureFlags: { build_warn_upcoming_system_version_change: true },
-      systemLogFile: systemLog.fd,
-      debug: false,
-    })
-    .runWithBuild()
-  t.true(normalizeOutput(output).includes('The Node.js version is 1.0.0 but the plugin "./plugin.js" requires >=1.0.0'))
-  const systemLogContents = await fs.readFile(systemLog.path, 'utf8')
-  await systemLog.cleanup()
-
-  t.true(systemLogContents.includes('plugin "./plugin.js" node support range includes v22'))
-})
-
```

Comment:
Should this be completely removed and not adjusted to have fixture have `engines.node` requiring higher version than 18.17 (? I'm not sure if I do read this test right) 

Also the `1.0.0` assertions tripped me out, but seems like that's because of version normalization - https://github.com/netlify/build/blob/6099258ec3d085a7e1e33042d435edf5880c8fdb/packages/testing/src/normalize.ts#L116-L117 
and the output of this test today is:
> The Node.js version is 16.14.0 but the plugin "./plugin.js" requires >=18.0.0
so I wonder wether instead we should do the bumps so node version IS in supported range, but plugin still require higher version than that

Reply:
This test is for the [functionality captured by the feature flag](https://github.com/netlify/build/pull/6223/files/f6722790ff241a262066123832df66358461af82#diff-d5a4df12b6188a8e04826328967795da04849c7943ffa467f33036a7d2eb3dacL83-L104) that I've opted to remove thus making this test non-functional. Open to re-adding the code there if we want to start evalutating the work needed to drop node 18 support.

Reply:
note that PR adding feature flag is the one that expanded this test (it also did change from snapshot to at least assertion on the user visible log which was great) in https://github.com/netlify/build/commit/b0aa3bf7e5babd7b3945000b4a10ec2ffb28531e#diff-6fc71606f82b3c3b3acd3320c55912d9c2b6d89b1d91a00c6e210a59c97e838dL106-R127 as this test was not just for the feature flag originally

Reply:
Restored the test in 81481a1